app/views/obituaries/index.html.erb

```diff
@@ -1 +1,26 @@
 <h1>All Obituaries</h1>
+  <table class="table table-striped table-bordered cart-table">
```

Comment:
This table will definitely need an upgrade in styling. Also, we have some decisions to make and discussions to have about what columns belong. This is likely to be basically the same view page as all search results so would be good to make it recyclable, perhaps with a partial.

Reply:
Nice, this looks great, and I agree it definitely makes sense to move it to a partial when we start recycling it.